src/kernel/xmlreader.c

```diff
@@ -1604,6 +1604,8 @@ static void parse_ai(race * rc, xmlNodePtr node)
         rc->flags |= RCF_MOVERANDOM;
     if (xml_bvalue(node, "learn", false))
         rc->flags |= RCF_LEARN;
+    if (xml_bvalue(node, "moveattack", false))
+        rc->flags |= RCF_ATTACK_MOVED;
```

Comment:
sehe ich das richtig, dass das hier eine Spezialregelung für Seeschlangen ist? Die können ja nicht bewachen, ehe sie attackieren, weil man den Ozean nicht bewachen darf, und deshalb sind die in 3.7 jetzt wohl kaputt?